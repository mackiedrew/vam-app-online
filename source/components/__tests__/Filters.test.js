import Subject from "../Filters";

describe("<Filters />", () => {
  describe("renders without crashing", () => {
    it("while closed", () => {
      shallow(<Subject />);
    });
    it("while open", () => {
      shallow(<Subject open />);
    });
  });

  describe("renders correctly", () => {
    it("while closed", () => {
      const tree = renderer.create(<Subject />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    it("while open", () => {
      const tree = renderer.create(<Subject open />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
