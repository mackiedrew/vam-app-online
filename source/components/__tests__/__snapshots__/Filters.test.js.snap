// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filters /> renders correctly while closed 1`] = `
<aside
  className="filters"
  style={
    Object {
      "marginLeft": "-100px",
    }
  }
>
  Filter!
</aside>
`;

exports[`<Filters /> renders correctly while open 1`] = `
<aside
  className="filters"
  style={
    Object {
      "marginLeft": "0px",
    }
  }
>
  Filter!
</aside>
`;
